src/formatter.ts

```diff
@@ -238,6 +238,10 @@ export class JupyterlabNotebookCodeFormatter extends JupyterlabCodeFormatter {
     }
     try {
       this.working = true;
+      if (this.getNotebookType() === null) {
+        this.working = false;
```

Comment:
Would it be useful to worth some kind of a message here in the console? In general I think that `this.getNotebookType()` result should be presented to the user when the default formatter cannot be found, whether it is because it is `null` or does not match the naming expected.

Reply:
The annoying thing about my problem case is that it occurs just as a brand new notebook is opened and an instant later, `getNotebookType()` is correctly populated and no longer null. I experimented with different ways to find this language as the JupyterLab environment knows it before the notebook is created, but they all seemed to have this same problem.

The reason I pursued this PR was because I was getting the "Jupyterlab Code Formatter Error" immediately upon creation, then nothing ended up being broken. I suppose need to do some more work defining replication because I'm not sure if it's just this library, or this library in combination with something else that's triggering it.

My thought was that the other cases where `this.getNotebookType()` are null seem to also suggest being run before the notebook is ready (e.g. `!this.notebookTracker.currentWidget` and `!metadata`).

Reply:
I decided to take another approach and have updated the PR, undoing all the changes from the original commit.